frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.scss

```diff
@@ -3,3 +3,7 @@
   padding-right: 30px;
```

Comment:
can you use PF spacer variables here?

Reply:
We should be able to. I'll take a look at it.

Reply:
Done!